contrib/great_expectations_semantic_types_expectations/great_expectations_semantic_types_expectations/expectations/expect_column_values_to_be_valid_mic.py

```diff
@@ -33,7 +33,7 @@ def is_valid_mic(mic_code: str, df) -> bool:
 class ColumnValuesToBeValidMic(ColumnMapMetricProvider):
 
     url = "https://www.iso20022.org/sites/default/files/ISO10383_MIC/ISO10383_MIC.csv"
-    df = pd.read_csv(url)
+    df = pd.read_csv(url, encoding="cp1250")
```

Comment:
What does this do?

Reply:
From discussion : this is the encoding of the specific CSV we are using in the test (was failing with `unicode`)

Reply:
That's the actual encoding of the file at that URL